interfaces/builtin/uio.go

```diff
@@ -100,6 +103,19 @@ func (iface *uioInterface) AppArmorConnectedPlug(spec *apparmor.Specification, p
 	// device node in /dev. Use AddDeduplicatedSnippet() for clarity
 	// in the resulting rules.
 	spec.AddDeduplicatedSnippet("/sys/devices/platform/**/uio/uio[0-9]** r,  # common rule for all uio connections")
+	uioConfigPath := "/sys/class/uio/" + iface.getUIOName(path) + "/device/config"
+	_, err = os.Stat(uioConfigPath)
+	if err != nil {
+		// This should not block the interface connection operation.
+		logger.Noticef("The following path does not exist %s", path)
```

Comment:
I wonder if this message needs to be a bit more informative - and shouldn't it use the `uioConfigPath` variable instead - perhaps something like 

```go
logger.Noticef("uio device config file does not exist %s", uioConfigPath)
```

Reply:
Thank you, I corrected this.